lib/l10n/intl_en.arb

```
@@ -557,6 +557,65 @@
+  "tradeInformation": "Trade Information",
+  "userInformation": "User Information",
+  "youAreSellingTo": "You are selling sats to",
+  "youAreBuyingFrom": "You are buying sats from",
+  "noMessagesYet": "No messages yet",
+  "today": "Today",
+  "yesterday": "Yesterday",
+  "youPrefix": "You: ",
+  "conversationsDescription": "Here you'll find your conversations with other users during trades.",
+  "orderId": "Order ID:",
+  "paymentMethod": "Payment Method:",
+  "createdOn": "Created on:",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)